app/controllers/error_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ErrorController < ApplicationController
+  skip_before_filter :ensure_current_user
+
+  def error
+  end
```

Comment:
In rails you don't need to implement an empty action. If you have a route that maps to error#error, rails will render app/view/error/error.... even if the error action is not there.